docs/developers/build-starrocks/build_starrocks_on_ubuntu.md

```diff
@@ -7,7 +7,7 @@ sudo apt-get update
 ```
 
 ```
-sudo apt-get install automake binutils-dev bison byacc ccache flex libiberty-dev libtool maven zip python3 python-is-python3 -y
+sudo apt-get install automake binutils-dev bison byacc ccache flex libiberty-dev libtool maven zip python3 python-is-python3 libbrotli1 -y
```

Comment:
You can refer to 
https://github.com/StarRocks/starrocks/blob/main/docker/dockerfiles/toolchains/toolchains-ubuntu.Dockerfile
Do you know why the libbrotli1 is needed?

Reply:
When building curl from scratch,  it requires libbrotli1. otherwise the build fails.

Given that in the Dockerfile, we have curl enrolled, we can skip this, and use the system's curl instead.

Reply:
I think it's a good idea to sync both apt lists